File: code/contests/div3Contest/1/D.cpp
#include <iostream>
using namespace std;

int can_vanya_win(int k) {
    int sum = 0;
    int count = 1;
    int digit_count = 0;

    while (digit_count < k) {
        int num = count;
        int digits[10];
        int len = 0;

        // Extract digits of num in reverse order
        while (num > 0) {
            digits[len++] = num % 10;
            num /= 10;
        }

        // Traverse digits from most significant to least
        for (int i = len - 1; i >= 0 && digit_count < k; --i) {
            sum += digits[i];
            digit_count++;
        }

        count++;
    }

    return sum;
}

int main() {
    int t;
    cin >> t;

    while (t--) {
        int k;
        cin >> k;
        cout << can_vanya_win(k) << endl;
    }

    return 0;
}

File: code/contests/div3Contest/1/D.py
#import math
def can_vanya_win(k):
    n_str = ''
    count = 1
    while len(n_str) != k:
        n_str+=str(count)
        count += 1

    sum = 0
    #n = int(n_str)

    for i in n_str:
        sum += int(i)

    return sum


t = int(input())
results = []

for _ in range(t):
    k = int(input())
    #can_vanya_win(n)
    results.append(can_vanya_win(k))

for res in results:
    print(res)